### PR TITLE
Separate init function

### PR DESCRIPTION
**Pull request by szepeviktor**
And reduce indentations!

See diffs w/o whitespaces: https://github.com/palmiak/timber-acf-wp-blocks/pull/36/files?diff=unified&w=1

### CODE REVIEW

File: timber-acf-wp-blocks.php [hunk 1]
```diff
@@ -7,198 +7,188 @@
...
 
-if ( ! function_exists( 'acf_register_block' ) ) {
-	return;
-}
-if ( ! function_exists( 'add_filter' ) ) {
-	return;
-}
-if ( ! function_exists( 'add_action' ) ) {
-	return;
+if ( function_exists( 'add_action' ) && function_exists( 'acf_register_block' ) ) {
```
Comment (szepeviktor) [inline timber-acf-wp-blocks.php:10]: `add_filter` is defined at the same time.

**Comment by palmiak** [comment]
Thanks @szepeviktor 

Much cleaner 👍 I'll double check it, but looks so much better.

**Comment by szepeviktor** [comment]
@palmiak Do you have the results of double-checking?

**Comment by palmiak** [comment]
Thank you and sorry it took so much time :)

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**